ui/controls.js

```diff
@@ -1985,9 +1985,6 @@ shaka.ui.Controls = class extends shaka.util.FakeEventTarget {
           this.seek_(this.player_.seekRange().end);
         }
         break;
-      case 'c':
-        this.player_.setTextTrackVisibility(!this.player_.isTextTrackVisible());
-        break;
```

Comment:
I think we don't want to lose this, so it should be implemented in another way :) 

Maybe something like:

```
if (this.player_.isTextTrackVisible()) {
  this.player_.selectTextTrack(null);
} else {
  this.player_.selectTextTrack(this.player_.getTextTracks()[0]);
}
```